<?xml version="1.0"?>
<launch>

    <!-- Mean filter for the raw value of wrench -->
    <node pkg="iris_cobot" type="wrench_filter" name="wrench_filter" output="screen" respawn="true"/>

    <!-- FT Theory Model -->
    <node pkg="iris_cobot" type="wrench_theory.py" name="wrench_theory" output="screen" respawn="true"/>

    <!-- Correction of raw values with theory model and off-line correction -->
    <node pkg="iris_cobot" type="wrench_correct.py" name="wrench_correct" output="screen" respawn="true"/>

    <!-- Force Torque to Velocity -->
    <node pkg="iris_cobot" type="wrench_to_vel" name="wrench_to_vel" output="screen" respawn="true"/>

    <!-- Weigh calculator -->
    <node pkg="iris_cobot" type="weight.py" name="weight" output="screen" respawn="true"/>

    <!-- Analysis of wrench based on time -->
    <!-- <node pkg="iris_cobot" type="time_series_analysis.py" name="time_series_analysis" output="screen" respawn="true"/> -->

</launch>